src/pages/Gateway/SnitchRow.tsx

```diff
@@ -1,17 +1,26 @@
 import { AoGatewayWithAddress } from '@ar.io/sdk/web';
+import Button from '@src/components/Button';
 import Dropdown from '@src/components/Dropdown';
-import { StatsArrowIcon } from '@src/components/icons';
+import { ReportsIcon, StatsArrowIcon } from '@src/components/icons';
 import Placeholder from '@src/components/Placeholder';
 import useEpochs from '@src/hooks/useEpochs';
 import useObserverToGatewayMap from '@src/hooks/useObserverToGatewayMap';
 import { useEffect, useState } from 'react';
-import { Link } from 'react-router-dom';
+import { Link, useNavigate } from 'react-router-dom';
+
+type ReportedOnByEntry = {
```

Comment:
nit: ReportedOnBy<who/address/gateway/observer>Entry - OnBy is a little confusing